recipe/meta.yaml

```diff
@@ -10,8 +10,7 @@ source:
   sha256: 1af9c3bb6f07f6266aeb038fef5a1fa63dafb70a466dcdfe44b6627e8d9aec59
 
 build:
-  number: 0
-  noarch: python
```

Comment:
It looks un-intuitive that we cannot declare noarch at the "top" level. But you clearly dug into this problem so I trust your solution.

Reply:
As far as I dug into it - it's some sort of complicated thing about individual outputs not inheriting everything from build. Either way, the way I'm doing it here is the way rust does it, so I'm going with the law of "bigger projects probably know more than I could"